### PR TITLE
Contrib bismark

### PR DESCRIPTION
**Pull request by tloizou**
Contributing Bismark

### CODE REVIEW

**Comment by boegel** [comment]
This also includes the commits from #989, should #989 should be merged in first, or this PR should be cleaned up.

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by tloizou** [comment]
@boegel: Fixed Bismark config. Ready for testing.

**Comment by tloizou** [comment]
@boegel: Sorry, ABINIT files were also included in this commit. I have now removed them and Bismark is ready for testing.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/eb07957191abb1597562 for a full test report.

File: easybuild/easyconfigs/b/Bismark/Bismark-0.10.1-goolf-1.4.10.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+name = 'Bismark'
+version = '0.10.1'
+
+homepage = 'http://www.bioinformatics.babraham.ac.uk/projects/bismark/'
+description = """A tool to map bisulfite converted sequence reads and 
+determine cytosine methylation states"""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = ["www.bioinformatics.babraham.ac.uk/projects/bismark/"]
```
Comment (boegel) [inline easybuild/easyconfigs/b/Bismark/Bismark-0.10.1-goolf-1.4.10.eb]: missing `http://` ?

**Comment by tloizou** [comment]
@boegel: I think it should be ok now! Please test.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/614c1f71b374835a66dc for a full test report.

**Comment by boegel** [comment]
OK, going in. Thanks @tloizou!

**Pull request merged**